internal/clients/team3/clientiigo.go

```diff
@@ -58,6 +63,19 @@ func (c *client) resetIIGOInfo() {
 	c.iigoInfo.ourDeclaredResources = 0
 }
 
+func (c *client) getOurRole() string {
+	if c.iigoInfo.startOfTurnJudgeID == shared.ClientID(3) {
```

Comment:
Instead of shared.ClientID(3), use `c.BaseClient.GetID()`, as it is more expandable in the future

Reply:
Sure, will do :)